index.bs

```diff
@@ -4940,8 +4940,8 @@ partial interface MLGraphBuilder {
   </summary>
     1. Let |desc| be a new {{MLOperandDescriptor}} initialized to |input|.{{MLOperand/[[descriptor]]}}.
     1. For |index| in [=the range=] 0 to |options|.{{MLResample2dOptions/axes}}'s [=list/size=], exclusive:
-        1. If |options|.{{MLResample2dOptions/sizes}} [=map/exists=], set |desc|'s [=MLOperand/shape=][|options|.{{MLResample2dOptions/axes}}[|index|]] to |options|.{{MLResample2dOptions/sizes}}[|index|] and return |desc|.
```

Comment:
If `MLOperand/shape` == `[[descriptor]].dimensions`, then isn't `MLOperand/shape` equivalent to `MLOperandDescriptor/dimensions` and fine to keep?

Reply:
👍